static-site/app/styles/globals.css

```diff
@@ -62,10 +62,6 @@ html {
   }
 }
 
-main {
-  padding: 0 25px;
-}
```

Comment:
Re: f9d2ebd9ec64fdc197219a928c9db5cc50bc1231

> The padding: 0 25px style that was applied to the `<main>` tag in global.css is not needed (or wanted!) on the index page, only on …every other page on the site.

Is this padding needed or wanted on the other pages? I took it out and can't tell a difference from a few spot checks. It would be cleaner to just remove the padding entirely and not need to add it back with the pageContent class applied everywhere, which seems fragile.

Reply:
dropped the commit.